db/sqlinterfaces.c

```diff
@@ -911,7 +927,9 @@ int sqlite3_open_serial(const char *filename, sqlite3 **ppDb,
                 */
             }
         }
-        comdb2_setup_authorizer_for_sqlite(*ppDb, &thd->authState, 1);
+        if (thd != NULL) {
```

Comment:
Fix for NULL "thd", part 1.  Setting up the authorizer for the database requires a non-NULL "thd".